mkosi/resources/mkosi.md

```diff
@@ -51,17 +53,21 @@ that may generate disk images with a number of bells and whistles.
 The following command line verbs are known:
 
 `summary`
-:   Outputs a human-readable summary of all options used for building an
+:   Output a human-readable summary of all options used for building an
```

Comment:
Might as well use `Show` instead of `Output`

Reply:
I did that, but then I noticed that the text needs more updating.